app/controllers/users_controller.rb

```diff
@@ -7,7 +7,7 @@ def create
     auth = request.env["omniauth.auth"]
     session[:access_token] = auth[:credentials][:token]
   
-  # UserFacade.create_user(auth[:info])
+  # UserFacade.find_or_create_user(auth[:info])
```

Comment:
I was wondering when we would use this! Totally makes sense, just was waiting for it!